### DISCUSSION TITLE
Suitable external power sources

### DISCUSSION THREAD

**Discussion by bbellina** [erichelgeson/BlueSCSI-v1#54] [Q&A] [answered]
I have a BlueSCSI 1.0c I am using with a Color Classic II on the SCSI port.  It seems to work well although it blocks a serial port.  I would like to also try it on my Mac Plus but understand that the Mac Plus SCSI port does not supply TERM power and so I need to have power supplied via the BERG (4 pronged connector on the opposite end from the SD card?) or the very tiny USB over the SD card. For each of these two options can you specify what kind of connector and what type of power supply would be required? I do not want to fry the thing. Thank you.

**Comment by erichelgeson** [comment, marked as answer]
You can use the micro USB connector on the blue pill or the berg connector. The plus won't have any molex or berg internally. Diodes are in place to prevent back power. Hope that helps let me know if you need more info.

**Comment by bbellina** [comment]
Thank you. Is there a particular voltage I should use to avoid exceeding the board’s tolerance?

Regards,

Brendan

**Reply by erichelgeson** [reply]
Micro USB supplies 5v. The berg connector uses the 5v pin.

**Comment by bbellina** [comment]
Thanks. Sounds like Kindle charger and cable should work fine then. I’ll order them and try it out with my Mac Plus. My external hard drive works for now but it won’t last forever. 

Regards,

Brendan
